README.md

```diff
@@ -35,9 +35,22 @@ CRD please refer to
 
 Custom Resources require Kubernetes >= 1.7
 
+For GPUs we recommend using Kubernetes 1.8
+
+ * Kubernetes 1.7 had some support for GPUs
+
 
 ## Installing the TfJob CRD and operator on your k8s cluster
 
+1. Ensure helm is running on your cluster
+
+   * On GKE with K8s 1.8, follow these
+     [instructions](https://github.com/kubernetes/helm/blob/master/docs/service_accounts.md)
```

Comment:
This is a better link instead which will be updated with the proper setup instructions: https://docs.helm.sh/using_helm/#tiller-namespaces-and-rbac